teuthology/task/install/util.py

```diff
@@ -125,3 +121,25 @@ def ship_utilities(ctx, config):
                 wait=False,
             ),
         )
+
+utilities_shipped = False
+
+@contextlib.contextmanager
+def ship_utilities(ctx, config):
+    """
+    Write a copy of valgrind.supp to each of the remote sites.  Set executables
+    used by Ceph in /usr/local/bin.  When finished (upon exit of the teuthology
+    run), remove these files.
+
+    :param ctx: Context
+    :param config: Configuration
+    """
+    assert config is None
+
+    global utilities_shipped
+    if utilities_shipped:
```

Comment:
minor, rename `utilities_shipped` to `do_ship_utilities` with defaults to True.
So code look like:
```
if do_ship_utilities:
   do_ship_utilities = False
   _ship_utilities(ctx, config)
else:
   log.info('Utilities already shipped, skip it...')
   yield
```

Reply:
Looks good will modify it. Thanks.